routes/api/project.rb

```diff
@@ -36,7 +36,7 @@ class CloverApi
         r.halt
       end
 
-      unless @project.user_ids.include?(@current_user.id)
+      unless @project.accounts.include?(@current_user)
```

Comment:
`include?` or more specifically equality is not safe for object comparisons, because they compare the objects not their ids. Here is a scary example;

```ruby
project = udec "...";
vm = project.vms.first;
project.vms.include?(vm)
=> true
project.reload.vms.include?(vm)
=> false
```

So we should stick with comparing ids.

Reply:
Here is a simpler example;

```ruby
Vm["b15336fb-7925-8374-962d-c1d746c7c7e1"] == Vm["b15336fb-7925-8374-962d-c1d746c7c7e1"]
=> false
```

Reply:
Then, why these work?

```ruby
1 == 1.0
=> true
"test" == "test"
=> true
```

Because their respective classes override `==` operator to make it more useful. It seems Sequel does not do that for its models, probably rightfully so.

Reply:
I changed it